symengine/lib/symengine_wrapper.pyx

```diff
@@ -689,7 +689,7 @@ cdef class Basic(object):
         elif (op == 5):
             return c2py(<RCP[const symengine.Basic]>(symengine.Ge(A.thisptr, B.thisptr)))
 
-    def expand(Basic self not None):
+    def expand(Basic self not None, *args, **kwargs):
```

Comment:
I could not find how `SymPy` handles complex expansion. I tried introducing keywords in `Pow` and the trigonometric functions, but the parser is throwing up issues. I think it would be better if these were handled later, as it would take up a lot of time right now. I'm dropping the commit. These are required only for a single import in `SymPy`.